### DISCUSSION TITLE
clair-mathcer and clair-indexer are going in restarting mode

### DISCUSSION THREAD

**Discussion by hima-Baghel** [quay/clair#1755] [Q&A]
When I try to deploy latest clair by cloning latest code from the given repo, clair-mathcer and clair-indexer container are going in restarting mode, when I check their logs an error is showing up "error : tls: failed to verify certificate: x509: certificate signed by unknown authority". Has anybody have any clue?

**Comment by hdonnay** [comment]
The services are being provided an invalid certificate:

    tls: failed to verify certificate: x509: certificate signed by unknown authority

**Reply by hima-Baghel** [reply]
Okay, Thanks
One more thing how can I get authorization token to access clair services?
As when I am trying to hit url "http://localhost:6060/indexer/api/v1/index_report/" got 401 status code in response. when I check clair-indexer logs I can see:"DBG failed to retrieve jwt from header component=middleware/auth/PSK.Check"
Anybody has any idea about it?

**Reply by hdonnay** [reply]
You need to sign a JWT with the same key and an issuer specified in the configuration.
